tests/StaticPersistenceTest.php

```diff
@@ -113,9 +113,9 @@ public function testFieldTypes()
             'name' => 'hello',
             'test_int' => 123,
             'test_float' => 123.45,
-            'test_date' => new \DateTime(),
-            'test_array' => ['a', 'b', 'c'],
-            'test_object' => new \DateInterval('P1Y'),
+            // 'test_date' => new \DateTime(),
```

Comment:
Array persistence no more support non-scalar data types?

Reply:
It fully does, but you must typecast, static persistence does not typecast seed data.